packages/web-app-files/src/components/AppBar/SelectedResources/BatchActions.vue

```diff
@@ -189,6 +238,61 @@ export default {
           })
         }
       })
+    },
+
+    acceptShares() {
+      this.triggerShareActions(shareStatus.accepted)
+    },
+
+    declineShares() {
+      this.triggerShareActions(shareStatus.declined)
+    },
+
+    triggerShareActions(newShareStatus) {
+      const errors = []
+      this.selectedFiles.forEach(async resource => {
+        try {
+          const share = await triggerShareAction(
+            resource,
+            newShareStatus,
+            !this.isOcis,
+            this.$client
+          )
+          if (share) {
+            this.UPDATE_RESOURCE(share)
+          }
+        } catch (error) {
+          errors.push(error)
+        }
+      })
+
+      if (errors.length === 0) {
+        this.SELECT_RESOURCES([])
+        return
+      }
+
+      console.log(errors)
```

Comment:
Dev leftover or on purpose?

Reply:
Intentional, because I don't want to swallow errors. Eventually we should come up with a proper logging solution.

Reply:
RFC ;)